### PR TITLE
[Vsphere] Add dimensions field mapping for datastore datastream to support TSDB.

### PR DESCRIPTION
**Pull request by ritalwar**
- Enhancement

## What does this PR do?

This PR adds dimension fields mapping for datastore data stream of Sphere to support TSDB enablement.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Related issues

- Relates #6266

### CODE REVIEW

File: packages/vsphere/docs/README.md [hunk 1]
```diff
@@ -390,6 +390,14 @@ An example event for `datastore` looks as following:
...
 |---|---|---|---|---|
 | @timestamp | Event timestamp. | date |  |  |
+| agent.id |  | keyword |  |  |
+| cloud.account.id | The cloud account or organization id used to identify different entities in a multi-tenant environment. Examples: AWS account id, Google Cloud ORG Id, or other unique identifier. | keyword |  |  |
+| cloud.availability_zone | Availability zone in which this host, resource, or service is located. | keyword |  |  |
+| cloud.instance.id | Instance ID of the host machine. | keyword |  |  |
+| cloud.project.id | The cloud project identifier. Examples: Google Cloud Project id, Azure Project id. | keyword |  |  |
+| cloud.provider | Name of the cloud provider. Example values are aws, azure, gcp, or digitalocean. | keyword |  |  |
+| cloud.region | Region in which this host, resource, or service is located. | keyword |  |  |
+| container.id | Unique container id. | keyword |  |  |
```
Comment (shmsr) [inline packages/vsphere/docs/README.md:400]:
```suggestion
| cloud.account.id | The cloud account or organization id used to identify different entities in a multi-tenant environment. Examples: AWS account id, Google Cloud org id, or other unique identifier. | keyword |  |  |
| cloud.availability_zone | Availability zone in which host, resource, or service is located. | keyword |  |  |
| cloud.instance.id | Instance ID of the host machine. | keyword |  |  |
| cloud.project.id | The cloud project identifier. Examples: Google Cloud Project id, Azure Project id. | keyword |  |  |
| cloud.provider | Name of the cloud provider. Examples: aws, azure, gcp, or digitalocean. | keyword |  |  |
| cloud.region | Region in which host, resource, or service is located. | keyword |  |  |
| container.id | Unique container id. | keyword |  |  |
```

File: packages/vsphere/changelog.yml [hunk 2]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.7.0"
+  changes:
+    - description: Added dimensions field mapping for datastore datastream to support TSDB.
```
Comment (shmsr) [inline packages/vsphere/changelog.yml:4]:
```suggestion
    - description: Add dimensions field mapping for datastore datastream to support TSDB.
```

Same reason: https://github.com/elastic/integrations/pull/6333#discussion_r1206407368

Comment (shmsr) [reply packages/vsphere/changelog.yml:4]: Also, PR title needs a similar update i.e., s/Added/Add

**Review by shmsr — CHANGES_REQUESTED** [review]

File: packages/vsphere/docs/README.md (hunk 1 above)
Comment (ritalwar) [reply packages/vsphere/docs/README.md:400]: This is auto-generated.

Comment (shmsr) [reply packages/vsphere/docs/README.md:400]: Ah, okay.

**Review by agithomas — APPROVED** [review]
LGTM!

**Review by shmsr — APPROVED** [review]
LGTM

**Pull request merged**